tests/generation/test_utils.py

```diff
@@ -2778,3 +2778,50 @@ def forward(self, input_ids, **kwargs):
         with self.assertRaises(TypeError):
             # FakeEncoder.forward() accepts **kwargs -> no filtering -> type error due to unexpected input "foo"
             bart_model.generate(input_ids, foo="bar")
+
+    @slow
+    def test_biased_generation(self):
```

Comment:
I wanted this to be an example in the docstring of the logits processor class... but we can't have slow examples there, correct?

For context, the model in `facebook/bart-large-cnn` is a file with 1.63GB

Reply:
> but we can't have slow examples there, correct?

Why? The testing of the docstrings should not hinder good examples. You can either put it in the example not in docstring syntax or use a flag so the test is being ignored altogether.